examples/ibm-is-ng/main.tf

```diff
@@ -706,3 +706,71 @@ data "ibm_is_placement_groups" "is_placement_groups" {
 ## List regions 
 data "ibm_is_regions" "regions" {
 }
+
+## Security Groups/Rules/Rule
+// Create is_security_groups data source
+data "ibm_is_security_groups" "example" {
+}
+
+// Create is_security_group_rule data source
+resource "ibm_is_vpc" "example" {
+  name = ibm_is_vpc.vpc1.name
+}
+
+resource "ibm_is_security_group" "example" {
+  name = "example-security-group"
+  vpc  = ibm_is_vpc.example.id
+}
+
+resource "ibm_is_security_group_rule" "example" {
+  depends_on = [
+      ibm_is_security_group.example,
+  ]
+  group     = ibm_is_security_group.example.id
+  direction = "inbound"
+  remote    = "127.0.0.1"
+  udp {
+    port_min = 805
+    port_max = 807
+  }
+}
+
+data "ibm_is_security_group_rule" "example" {
+  depends_on = [
+      ibm_is_security_group_rule.example,
+  ]
+    security_group_rule = ibm_is_security_group_rule.example.rule_id
+    security_group = ibm_is_security_group.example.id
+}
+
+// Create is_security_group_rules data source
+resource "ibm_is_vpc" "example" {
+  name = ibm_is_vpc.vpc1.name
+}
+
+resource "ibm_is_security_group" "example" {
+  name = "example-security-group"
+  vpc  = ibm_is_vpc.example.id
+  depends_on = [
+    ibm_is_vpc.example,
+  ]
+}
+
+resource "ibm_is_security_group_rule" "example" {
+  group     = ibm_is_security_group.example.id
+  direction = "outbound"
+  remote    = "127.0.0.1"
+  tcp {
+    port_min = 8080
+    port_max = 8080
+  }
+  depends_on = [
+    ibm_is_security_group.example,
+  ]
+}
+data "ibm_is_security_group_rules" "example" {
+  depends_on = [
+    ibm_is_security_group_rule.example,
+  ]
+  security_group = ibm_is_security_group.example.id
+}
```

Comment:
I remember we used to add only the resource and datasource of the feature that we are developing and we refer to the existing resource.. like attach the existing VPC and Security Group to the rule..

Reply:
done.

Reply:
done.